linters/pragma-once/test/__snapshots__/pragma-once_basic_fmt.shot

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
If desired, I can move the filepath to not nest this inside of __snapshots__ so that it's more apparent, although I think having autogenerated files in a subdirectory (as is right now) is probably the better move.

Reply:
agree w you